lakeshore336/system_tests/test-reports/TEST-tests.lksh336.Lksh336Tests-20220801091438.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you add your `test-reports` folder to `.gitignore` and then remove this from git?